userbenchmark/gnn-pyg/__init__.py

```diff
@@ -0,0 +1,53 @@
+import torch
```

Comment:
Do we plan to maintain the `gnn-pyg` userbenchmark for long term? or it can be merged into our planed `cpu` userbenchmark together? BTW, for those new added GNN models, can we test them with `run.py` also?

Reply:
When `cpu` userbenchmark starts to use, this `gnn-pyg` can be moved inside.
I have tested these GNN models in `run.py`.

Reply:
Yes I agree, I suggest we use a single `cpu` userbenchmark to handle all CI workflows.